application/src/main/java/org/opentripplanner/service/vehiclerental/model/VehicleRentalStation.java

```diff
@@ -224,6 +392,86 @@ private List<RentalVehicleTypeCount> vehicleRentalTypeMapToList(
       .entrySet()
       .stream()
       .map(vtc -> new RentalVehicleTypeCount(vtc.getKey(), vtc.getValue()))
+      // we sort to have reproducible results in tests
+      .sorted(Comparator.comparing(count -> count.vehicleType().id().toString()))
```

Comment:
Could this be simplified as `Comparator.comparing(RentalVehicleTypeCount::vehicleType)`?